lib/ReactViews/Map/Panels/SharePanel/SharePanelContent.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { FC, useState, useRef, useMemo, useCallback } from "react";
+import { useTranslation } from "react-i18next";
+
+import Terria from "../../../../Models/Terria";
+import ViewState from "../../../../ReactViewModels/ViewState";
+
+import Box from "../../../../Styled/Box";
+import Spacing from "../../../../Styled/Spacing";
+import Text from "../../../../Styled/Text";
+
+import { ShareUrl, IShareUrlRef, ShareUrlBookmark } from "./ShareUrl";
+import { canShorten } from "./BuildShareLink";
+import { AdvancedOptions } from "./AdvancedOptions";
+import { StyledHr } from "./StyledHr";
+import { PrintSection } from "./Print/PrintSection";
+import { useCallbackRef } from "../../../useCallbackRef";
+
+interface ISharePanelContentProps {
+  terria: Terria;
+  viewState: ViewState;
+  closePanel: () => void;
+}
+
+export const SharePanelContent: FC<ISharePanelContentProps> = ({
```

Comment:
I think its confusing to have this separate to SharePanel. Especially as the content for StoryShare and CatalogShare are rendered directly in SharePanel and dont have separate components. Im sure there is a good reason though, just not the clearest structure for maintainability going forward